test/percy/appium/driverWrapper.test.mjs

```diff
@@ -47,5 +48,33 @@ describe('AppiumDriver', () => {
       );
       expect(await driver.getPercyOptions()).toEqual({ enabled: false, ignoreErrors: true });
     });
+
+    it('should return an element by XPath for wd driver', async () => {
+      const driver = new AppiumDriver(wdDriver());
+      const xpath = '//div[@class="example"]';
+      const element = await driver.elementByXPath(xpath);
+      expect(element).toBe('element');
+    });
+
+    it('should return an element by id for wd driver', async () => {
+      const driver = new AppiumDriver(wdDriver());
+      const id = 'some id';
+      const element = await driver.elementByAccessibilityId(id);
+      expect(element).toBe('element');
```

Comment:
add an expectation on what function is called with what args

Reply:
done